example-llm-workflows/README.md

```diff
@@ -0,0 +1,483 @@
+
+Let's first unzip [our tar file][tar-link] containing binaries and what's needed to run our demos, including workflow JSON files and [LLaMA 3][llama-3] models (created from [llama.cpp][llama.cpp]).
+
+```console
+tar -xf haii.tar.gz -C <target/directory>
+cd <target/directory>/example-llm-workflows
+```
+
+Everything going forward should take place within the `example-llm-workflows` folder.
+
+In one window, let's run the forked Homestar node (built with LLaMA 3 bindings) while also setting some environment variables:
+
+```console
+EVERY_CLI=true LLAMA_METAL=1 ./homestar start -c homestar_config.toml
+```
+
+In another console window, let's run the local version of EVERY-CLI:
+
+```console
+./every-cli/cli.js dev ../example-llm-workflows/llm.wasm
+```
+
+After running the above command, you should see something similar to this:
+
+```console
+IPFS is running at http://127.0.0.1:5001/debug/vars
+✔ Functions parsed and compiled
+✔ Homestar is running at http://127.0.0.1:8020
+✔ Control Panel is running at https://control.everywhere.computer
+
+[9:40:54 AM]  WARN  [listhen] Trying to listhen on private host "127.0.0.1" with public option enabled.
+
+[9:40:54 AM] ◐ Starting cloudflared tunnel to http://127.0.0.1:3000/
+
+              █▀▀▀▀▀▀▀██▀██████▀██▀▀▀▀▀██▀▀▀▀▀▀▀█
+              █ █▀▀▀█ █▄ ▀ ▄▄▀▄▀▄ █  ████ █▀▀▀█ █
+              █ █   █ █▄▄▀▄▀ ▄██  ▀▀▀   █ █   █ █
+              █ ▀▀▀▀▀ █ █ █▀▄ ▄ ▄▀█▀█▀█ █ ▀▀▀▀▀ █
+              █▀▀▀▀▀█▀▀▀▀▀█ ▀  ▄▀▄█ ▄▀▀▀▀█▀█▀█▀██
+              █▄█ ▀▀ ▀▄▄ █▀▀▄▀▀▄██▄▄▀█▄██▄██▀▄ ▄█
+              █▄ ██▀▀▀▀█ ▄ ██▄▀▀▀▄▄▀▄ █  ▄▀▀▀▄███
+              █▀▀▄ █▀▀▀▀▄ █▄ ▀▀ ▀▀▄ ▄█▄█ ▄▀▀██ ▄█
+              █ ▀▀▀▀▄▀█▀█▀█ ▄▀ █▀▄█▄▄ ▄ ██▄▀█▄▀██
+              █▀█▀ ▀█▀▀▀▀▄▀▀██▀▄▄▀▄▄▀█▄█▀ ▀▀ █ ▄█
+              █  ▄██▀▀██▄▀ ██▄▄▄▀█▄ ▄ ▀  █▄▀█▄▀██
+              █ █ ██▀▀███▀█▄  ▀▄▀ ▄▄█▀▄▀ ▄█▀ █ ▄█
+              █ ██ ▀█▀ █  █ ▄▀▀▄▀█ ▄▄▀█ ▀▀▀ ▄▄█▀█
+              █▀▀▀▀▀▀▀█  █▀▀██▀▄▄▄▄ ▄▀  █▀█ ▀▀▀▄█
+              █ █▀▀▀█ █▀▀█ ██▄ ▄▀▄ ▄▄▄▄ ▀▀▀ ▄▄▀ █
+              █ █   █ █  ▀█▄  ▀  ▄▄ ▀▀ █ ▀▄▄▄█▄▄█
+              █ ▀▀▀▀▀ █ █ █ ▄▀  ▀█▄█▄ ▀▀█▀▀▄▄▄▀██
+              ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+
+  ➜ Local:    http://127.0.0.1:3000/
+```
+
+EveryCLI has started a gateway that loads Wasm components onto [IPFS][ipfs] for persistent storage, prepares workflows, and integrates with the Homestar runtime to schedule and execute these workflows.
+
+With EveryCLI up, in another window, let's POST a job/workflow to the local EveryCLI `localhost` endpoint, which will execute the [`simple.json`](./simple.json2) workflow on [Homestar][homestar]:
+
+```console
+curl localhost:3000/run --json @simple.json
+```
+
+Once this completes (which can take some time mind you), we'll get back a response like this from the execution, which is returning text from a [LLaMA 3 instruction tuned model][instruction-llama]:
```

Comment:
Might be good to show the prompt here to contextualize the returned text.

--- 
I see you have the prompt below, but mentioning "Attractions in Pittsburgh" here would still be good.